dapps/marketplace/src/components/TelegramLoginButton.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, memo } from 'react'
+
+const TelegramLoginButton = ({ redirectURL, buttonText, className }) => {
+  let rootEl
+
+  useEffect(() => {
+    const script = document.createElement('script')
+    script.async = true
+    script.setAttribute('src', 'https://telegram.org/js/telegram-widget.js?7')
```

Comment:
curious what is the `?7` for?

Reply:
I'm not sure either. That URL is from here: https://core.telegram.org/widgets/login